.github/ISSUE_TEMPLATE/security-vulnerability.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Security Vulnerability
+about: Report vulnerability in the project
+---
+
+<!-- Thank you for your contribution. Before you submit the issue:
+1. Search open and closed issues for duplicates.
+2. Read the contributing guidelines.
+3. In case of vulnerabilities with a CVSS Base Score of *7.0* or above (see risk assessment section), please send an email directly to security@kyma-project.io instead of using this issue tracker. You can still use this template in your email. You can also use email for vulnerabilities with lower severity, if you prefer to keep your report confidential.
```

Comment:
For sure we need to update the email as we changed it: 
kyma-security@googlegroups.com